docker/build-tools/prow-entrypoint.sh

```diff
@@ -29,8 +29,8 @@ function read_gcp_secrets() {
   for item in "${secrets[@]}"; do
     proj="$(<<<"$item" jq .project -r)"
     secret="$(<<<"$item" jq .secret -r)"
-    env="$(<<<"$item" jq .env -r // "")"
-    file="$(<<<"$item" jq .file -r // "")"
+    env="$(<<<"$item" jq -r '.env // ""')"
```

Comment:
Does the -r need to be inside the single quotes or not there? Some quick testing:
```
> jq '.columns.idd // ""' file
""
>  jq -r '.columns.idd // ""' file
```
Not sure if you want the "" default

Reply:
No we want it empty - otherwise we will try to write to the file `""`